src/App.js

```diff
@@ -1,25 +1,20 @@
 import React from 'react';
-import logo from './logo.svg';
 import './App.css';
+import ItemRow from './components/ItemRow'
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div className="container">
+      <div className="App">
+        <h1>WOB</h1>
+          <ItemRow index = {1}/>
```

Comment:
Maybe `index = 1` would be enough.
We use `{}` to put expressions or javascript command in JSX.
Here it describes a similar situation but with strings https://reactjs.org/docs/jsx-in-depth.html#string-literals

Reply:
Got it. 
But as the documentation says the difference is irrelevant. 
But your comment rises up the question about our project's own code conventions.  Would it have it? It's your call Boss... :)

Reply:
Parsing error: JSX value should be either an expression or a quoted JSX text
   8 |       <div className="App">
   9 |         <h1>WOB</h1>
> 10 |           <ItemRow index = 1/>
     |                            ^
  11 |           <ItemRow index = {2}/>
  12 |         <div>
  13 |           <button>Compare</button>eslint

Reply:
"
Parsing error: JSX value should be either an expression or a quoted JSX text
   8 |       <div className="App">
   9 |         <h1>WOB</h1>
> 10 |           <ItemRow index = 1/>
     |                            ^
  11 |           <ItemRow index = {2}/>
  12 |         <div>
  13 |           <button>Compare</button>eslint
"

Reply:
Parsing error: JSX value should be either an expression or a quoted JSX text
Can't change, Boss..